api/src/main/scala/hmda/api/http/HmdaCustomDirectives.scala

```diff
@@ -17,25 +16,32 @@ trait HmdaCustomDirectives extends ApiErrorProtocol {
     RejectionHandler.newBuilder()
       .handle {
         case AuthorizationFailedRejection =>
-          val errorResponse = ErrorResponse(403, "Unauthorized Access", "")
-          complete(ToResponseMarshallable(StatusCodes.Forbidden -> errorResponse))
+          extractUri { uri =>
+            val errorResponse = ErrorResponse(403, "Unauthorized Access", uri.toString)
```

Comment:
This is going to result in the full URI being put into the `path` attribute, while the rest of the routes look like they're using a manually constructed path string, and passing that into all errors.  I think we should make this consistent.

I personally prefer having the full URI, and I also think we could replace all of the manual path building with a single directive or error handler.  That seems out outside the scope of this PR, though.  Perhaps for now, we just do a `uri.path.toString` for now to make it consistent?